program5.js

```diff
@@ -0,0 +1,13 @@
+var fs = require("fs");
+var path = require("path");
+
+var file = process.argv[2];
+var ext = process.argv[3];
+
+fs.readdir(file, function(err, list) {
```

Comment:
```suggestion
fs.readdir(file, (err, list) => {
```
アロー関数か普通の関数か統一したほうが良さそう